form_test/server.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask, render_template, request, redirect
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+  return render_template("index.html")
+
+@app.route('/result', methods=['POST'])
+def create_user():
+   print "Got Post Info"
+   name = request.form['name']
+   location = request.form['location']
+   language = request.form['language']
+   comment = request.form['comment']
+   return render_template("result.html")
```

Comment:
After you store the user-entered values in these local variables, what happens to them? From the code, it doesn't look as though they'll b displayed on the resulting page yet =O!